lib/main_common.pm

```diff
@@ -1734,6 +1735,11 @@ sub load_security_tests_crypt {
     loadtest "console/consoletest_finish";
 }
 
+# Other security tests other than FIPS
+sub load_security_tests_apparmor_status {
+    loadtest "security/apparmor/aa_status";
+}
+
```

Comment:
Please keep the functions which are only used in SLE within the sle/main but I would very much prefer if you could execute the same test on openSUSE and then of course keep this here. Especially the apparmore case might be a good candidate to start with.

Reply:
I plan to add it to openSUSE in future ( change opensuse/main.pm)

Reply:
Done